tests/unit/Codeception/Module/Db/DbCleaningTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+abstract class DbCleaningTest extends \PHPUnit_Framework_TestCase
```

Comment:
Very good suggestion. 

Also we need MySQL tests there as well, and Postgress too...